janitor/functions.py

```diff
@@ -546,16 +546,12 @@ def fill_empty(df, columns, value):
         for col in columns:
             assert (
                 col in df.columns
-            ), "{col} missing from dataframe columns!".format(
-                col=col
-            )
+            ), "{col} missing from dataframe columns!".format(col=col)
```

Comment:
We might be having black formatting clashes here. Can I check, are you using `black -l 79`? Or is there something else that's causing this? I noticed that between PRs, this flip-flops in formatting.

Reply:
Just purely running `make check` which runs `black -l 79 .`

Reply:
huh, dunno then. No worries.

Reply:
_shrug_